tools/donate-cpu.py

```diff
@@ -95,6 +97,21 @@
         print('Unhandled argument: ' + arg)
         sys.exit(1)
 
+if sys.version_info.major < 3:
+    print("#" * 80)
+    print("#" * 80)
+    print("IMPORTANT")
+    print("Please consider switching to Python 3.")
+    print("")
+    print("We plan to completely drop Python 2 support")
+    print("for the Donate CPU client in the near future.")
```

Comment:
Do we want to be more specific than "near future"? Like "after jan 1 2020" or "at the release of cppcheck 1.91" (or whatever is a good time)? Or maybe it's good to keep it open so we can change it whenever we want...

Reply:
My idea was to use "near future" until we have agreed on a specific date or so. It is quickly changed when a date is known.